.github/workflows/frontend_tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: Run Frontend Tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
```

Comment:
nit: same as above; my apologies for my unfamiliarity if this is the correct styling